tasks/actorstate/task.go

```diff
@@ -29,12 +29,15 @@ type Task struct {
 	closer lens.APICloser
 
 	extracterMap ActorExtractorMap
+
+	models model.PersistableList
```

Comment:
ActorState tasks are now passed a list of models to run extraction for during processing.